main.py

```diff
@@ -0,0 +1,116 @@
+from Board import Board
+from Snake import Snake
+from Ladder import Ladder
+
+import random
+
+def roll_die():
```

Comment:
These type of methods can be created as a static method. Also shift it to player class.